entities/src/revert_log.rs

```diff
@@ -13,7 +12,7 @@ pub struct Model {
     pub rpc_key_id: u64,
     pub timestamp: DateTimeUtc,
     pub method: Method,
-    #[serde(serialize_with = "serialization::vec_as_address")]
```

Comment:
Same question as before. Also, is Vec<u8> intended? before, it was marked as a uuid